$OpenBSD$

We know what the full path of the binary will be.

Index: src/fileutil.cc
--- src/fileutil.cc.orig
+++ src/fileutil.cc
@@ -150,7 +150,8 @@ void GetExecutablePath(string* path) {
   mypath[size] = 0;
   *path = mypath;
 #else
-#error "Unsupported OS"
+  char mypath[PATH_MAX + 1] = "TRUEPREFIX/bin/ckati";
+  *path = mypath;
 #endif
 }
 
